routes/project.rb

```diff
@@ -24,6 +24,17 @@ class Clover
     end
 
     r.post true do
+      if current_account.projects_dataset.count >= 10
+        err_msg = "Project limit exceeded. You can create up to 10 projects. Contact support@ubicloud.com if you need more."
+        if api?
+          fail CloverError.new(400, "InvalidRequest", err_msg)
+        else
+          no_authorization_needed
```

Comment:
```suggestion
        no_authorization_needed
        if api?
          fail CloverError.new(400, "InvalidRequest", err_msg)
        else
```

I'm considering changing the authorization check to also happen even if the error handler reraises the error.